tool/src/main/java/org/datacommons/tool/Processor.java

```diff
@@ -71,17 +72,16 @@ public static Integer process(Args args) throws IOException {
 
         // Having looked up the external IDs, resolve the instances.
         logger.info("Resolving Instance MCF nodes");
-        Mcf.McfGraph resolvedGraph = processor.resolveNodes();
-
-        // Add stats from resolved graph to statChecker.
-        processor.addStats(List.of(resolvedGraph));
+        nodesForStatProcessing = List.of(processor.resolveNodes());
 
         // Resolution for table nodes will happen inside processTables().
-      } else {
-        // Add stats from graphs from nodesForVariousChecks to statChecker.
-        processor.addStats(processor.nodesForVariousChecks);
       }
 
+      // Extract relevant nodes from instance MCFs and add to statChecker.
+      processor.addStats(nodesForStatProcessing);
+      // Check svObs nodes from instanceMCFs for value inconsistencies.
+      processor.checkSvObInconsistencies(nodesForStatProcessing);
```

Comment:
we loop over the nodes twice... should we combine these both into a processStats() helper?

Reply:
done